src/Repository/QuizTemplateRepository.php

```diff
@@ -3,39 +3,24 @@
 
 namespace QuizApp\Repository;
 
+use ReallyOrm\Criteria\Criteria;
 use ReallyOrm\Repository\AbstractRepository;
 
 class QuizTemplateRepository extends AbstractRepository
 {
-    public function getTableName(): string
-    {
-        return 'quiz_template';
-    }
-
-    public function getNumberOfQuizzes(): int
-    {
-        $sql = 'SELECT count(*) as quizzesNumber FROM quiz_template';
-        $dbStmt = $this->pdo->prepare($sql);
-        $dbStmt->execute();
-
-        return $dbStmt->fetch(\PDO::FETCH_COLUMN);
-    }
-
     public function saveQuestionsForQuiz($id, $questions)
```

Comment:
Let us have some consistency for variable names please. Choose one name for statements ($dbStmt or $sqlStm) and use it throughout. 
In a future iteration when making multiple calls to the database which are co-dependent please consider using transactions. 
https://www.php.net/manual/en/pdo.begintransaction.php
https://www.php.net/manual/en/pdo.commit.php
https://www.php.net/manual/en/pdo.rollback.php